python/ray/worker.py

```diff
@@ -1495,6 +1488,32 @@ def shutdown(exiting_interpreter=False):
     global_worker.set_mode(None)
 
 
+def exit_actor():
+    """Exiting the current actor.
+
+    This function is used to disconnect an actor and exit the worker.
+    TODO(guoyuhong): The actor should not be reconstructed after this
+    function is called even when max_reconstructions is not 0.
```

Comment:
Why is this a TODO? Isn't this already implemented?

Reply:
@raulchen  In my understanding, this is implemented in our repro not here, right?

Reply:
I mean there is not argument in `worker.raylet_client.disconnect()` to mark this is intentional disconnect. Maybe `worker.raylet_client.disconnect()` will not trigger the actor FO?

Reply:
@raulchen It's my misunderstanding and I have removed the `TODO`.